host_core/native/hostcore_wasmcloud_native/src/wasmruntime.rs

```diff
@@ -103,10 +151,17 @@ pub fn on_load(env: Env) -> bool {
 #[rustler::nif(name = "runtime_new")]
 pub fn new<'a>(
     env: rustler::Env<'a>,
-    _config: ExRuntimeConfig,
+    ExRuntimeConfig { host_id }: ExRuntimeConfig,
 ) -> Result<ResourceArc<RuntimeResource>, rustler::Error> {
-    let host_handler = ElixirHandler { pid: env.pid() };
-    let rt = WcRuntime::from_host_handler(host_handler)
+    let _ = env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("warn"))
```

Comment:
super tiny nit. Clippy now rejects this kind of syntax as a bad idiom. In a case when you truly don't care about the outcome, I _think_ the current accepted idiom is the "ok trick", e.g.

```
env_logger::Builder::from_env(....).format_timestamp(..).try_init().ok();
```
I'll defer to @thomastaylor312 or @brooksmtownsend on idiomatic Rust here, though. I just know that a number of my own reviews have been flagged for using the `let _ = foo` pattern.

Reply:
No clippy warnings here, but there are plenty other ones:

```
$ cargo clippy --all-targets                                   
    Checking hostcore_wasmcloud_native v0.1.2 (/home/rvolosatovs/src/github.com/wasmCloud/wasmcloud-otp/host_core/native/hostcore_wasmcloud_native)
warning: unused imports: `Atom`, `Binary`, `Encoder`, `Error`, `ListIterator`, `MapIterator`, `OwnedEnv`, `Term`, `resource::ResourceArc`, `types::tuple`
 --> src/environment.rs:4:5
  |
4 |     resource::ResourceArc, types::tuple, Atom, Binary, Encoder, Env, Error, ListIterator,
  |     ^^^^^^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^  ^^^^  ^^^^^^  ^^^^^^^       ^^^^^  ^^^^^^^^^^^^
5 |     MapIterator, OwnedEnv, Term,
  |     ^^^^^^^^^^^  ^^^^^^^^  ^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: this async expression only awaits a single future
   --> src/wasmruntime.rs:244:24
    |
244 |         TOKIO.block_on(async { component.actor.call(operation.clone(), Some(payload)).await });
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: you can reduce it to: `component.actor.call(operation.clone(), Some(payload))`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_async_block
    = note: `#[warn(clippy::redundant_async_block)]` on by default

warning: unused variable: `env`
   --> src/wasmruntime.rs:182:5
    |
182 |     env: rustler::Env<'a>,
    |     ^^^ help: if this is intentional, prefix it with an underscore: `_env`
    |
    = note: `#[warn(unused_variables)]` on by default

warning: function `spawn` is never used
  --> src/lib.rs:45:15
   |
45 | pub(crate) fn spawn<T>(task: T) -> JoinHandle<T::Output>
   |               ^^^^^
   |
   = note: `#[warn(dead_code)]` on by default

warning: redundant clone
   --> src/inv.rs:360:30
    |
360 |         let mut bad_inv = inv.clone();
    |                              ^^^^^^^^ help: remove this
    |
note: cloned value is neither consumed nor mutated
   --> src/inv.rs:360:27
    |
360 |         let mut bad_inv = inv.clone();
    |                           ^^^^^^^^^^^
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_clone
    = note: `#[warn(clippy::redundant_clone)]` on by default

warning: redundant clone
   --> src/inv.rs:367:37
    |
367 |         let mut really_bad_inv = inv.clone();
    |                                     ^^^^^^^^ help: remove this
    |
note: cloned value is neither consumed nor mutated
   --> src/inv.rs:367:34
    |
367 |         let mut really_bad_inv = inv.clone();
    |                                  ^^^^^^^^^^^
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_clone

warning: the following explicit lifetimes could be elided: 'a
   --> src/wasmruntime.rs:152:1
    |
152 | / pub fn new<'a>(
153 | |     env: rustler::Env<'a>,
154 | |     ExRuntimeConfig { host_id }: ExRuntimeConfig,
155 | | ) -> Result<ResourceArc<RuntimeResource>, rustler::Error> {
    | |_________________________________________________________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_lifetimes
    = note: `#[warn(clippy::needless_lifetimes)]` on by default
help: elide the lifetimes
    |
152 ~ pub fn new(
153 ~     env: rustler::Env<'_>,
    |

warning: the following explicit lifetimes could be elided: 'a
   --> src/wasmruntime.rs:286:1
    |
286 | / pub fn receive_callback_result<'a>(
287 | |     token_resource: ResourceArc<CallbackTokenResource>,
288 | |     success: bool,
289 | |     binary_result: Binary<'a>,
290 | | ) -> NifResult<rustler::Atom> {
    | |_____________________________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_lifetimes
help: elide the lifetimes
    |
286 ~ pub fn receive_callback_result(
287 |     token_resource: ResourceArc<CallbackTokenResource>,
288 |     success: bool,
289 ~     binary_result: Binary<'_>,
    |

warning: the borrowed expression implements the required traits
   --> src/lib.rs:461:48
    |
461 |     let extracted = match wasm::extract_claims(&bytes) {
    |                                                ^^^^^^ help: change this to: `bytes`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow
    = note: `#[warn(clippy::needless_borrow)]` on by default

warning: `hostcore_wasmcloud_native` (lib test) generated 9 warnings (7 duplicates) (run `cargo clippy --fix --lib -p hostcore_wasmcloud_native --tests` to apply 2 suggestions)
warning: `hostcore_wasmcloud_native` (lib) generated 7 warnings (run `cargo clippy --fix --lib -p hostcore_wasmcloud_native` to apply 6 suggestions)
    Finished dev [unoptimized + debuginfo] target(s) in 2.48s
```